qutebrowser/config/configdata.yml

```diff
@@ -656,6 +656,11 @@ completion.web_history_max_items:
 
     0: no history / -1: unlimited
 
+completion.use_best_match:
+  type: Bool
+  default: true
```

Comment:
I think it'd be better to default this to `false`, to keep the existing behavior.